### ISSUE TITLE
Q: benefits in explicit server name for local assets?

### ISSUE THREAD

**Issue by bard** [idleberg/Bootstrap-Listr#261]
I was wondering if there are any benefits in generating asset refs containing protocol/host when  assets are local. In other words if this:

```
  <link rel="stylesheet" href="http:///content/assets/css/font-awesome.min.css" />
```

...is better than this:

```




https://github.com/idleberg/Bootstrap-Listr/blob/274731a8aa71581c3f070e0dd8604fabbd6dc771/src/listr-functions.php#L70

**Comment by idleberg** [comment]
If I got this right, you would love to specify relative URLs to your assets? I *think* that you currently have to specify absolute URLs to overcome the forwarder (which was contributed, so I can't say for sure!) I will look into it when I have time for it.

**Comment by bard** [comment]
Yes, correct, relative URLs to assets would be my ideal target scenario.
